app/src/utils/session.js

```diff
@@ -24,6 +25,7 @@ class SessionManager {
     console.log('Update session:', session)
     this.session.value = session
     this.graph.value = new FileGraph(session.graph)
+    this.hint.value = ''
```

Comment:
hints should be a list ([String]), add new one instead of replacement

Reply:
What do you mean add a new one? Then how to manage that?